cmd/create/externalauthprovider/cmd.go

```diff
@@ -90,7 +90,9 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command, argv []string) error {
 		return err
 	}
 
-	r.Reporter.Infof("Successfully created an external authentication provider for cluster '%s'", cluster.ID())
+	r.Reporter.Infof("Successfully created an external authentication provider for cluster '%s'. "+
+		"It can take a while for the creation of an external authentication provider to become fully effective.",
```

Comment:
"a while" doesn't give me much idea about how long to wait. Maybe try to give the user a better idea with wording like: "a few minutes", "up to 30 minutes", ...

Reply:
Thanks @philipwu08 , I'll go with a few minutes then

Reply:
@philipwu08 Updated, would you mind taking another look?